client/package.json

```diff
@@ -252,7 +252,6 @@
     }
   },
   "scripts": {
-    "test": "npm run test:grammar",
```

Comment:
I think it was good. It is intended to have a way to launch all the tests of the client at the same time, even if for now there is only test:grammar. It also give us a way to uniformly call the tests of the client and the server just with "npm test --workspaces" (thought the server has no tests for the moment).

Reply:
Done